TechDraw/TechDrawTools.FCMacro

```diff
@@ -1,4 +1,4 @@
-from PySide2 import QtCore, QtGui, QtWidgets # PySide2
+from PySide2 import QtCore, QtGui, QtWidgets # PySide package is FreeCAD's specific package
```

Comment:
Sorry to bother you about this but `PySide2` is NOT FreeCAD specific, only `PySide` is. Try `PySide.__file__` from FreeCAD's Python Console to convince yourself.

Reply:
To use PySide2 instead of PySide was not my idea. I changed the program code following a recommendation of @openBrain see : https://forum.freecadweb.org/viewtopic.php?f=35&t=55029&start=30.
The macro is working, it was downloaded over 120 times, nobody told problems when using PySide2.
Please tell me the lines I shall enter, so that you can merge the PR.